libs/langchain-openai/src/utils/azure.ts

```diff
@@ -3,7 +3,7 @@ export interface OpenAIEndpointConfig {
   azureOpenAIApiInstanceName?: string;
   azureOpenAIApiKey?: string;
   azureOpenAIBasePath?: string;
-  baseURL?: string;
+  baseURL?: string | null;
```

Comment:
This doesn't look necessary?

Reply:
It's because of this change: https://github.com/openai/openai-node/commit/eb82824f89c31d752cdb4f198716a199e3b6eafd#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R26

But maybe it should be dealt with differently.